ap_src/ap_app/utils/switch_permissions.py

```diff
@@ -0,0 +1,68 @@
+# Models
+from ..models import (UserProfile)
+from django.contrib.auth.models import User
+
+# Utils
+from teams_utils import get_users_sharing_teams
+from user_profile import get_user_id_from_username
```

Comment:
You can group both imports from user_profile into one line:

`from user_profile import get_user_id_from_username, get_userprofile_id_from_user_id`

Reply:
The multiple imports here were to avoid using wildcard imports as importing specific functions feels like a better practice (in my opinion).

The import will be changed to just use one line.